extensions/aggregate_functions.yaml

```diff
@@ -2,49 +2,59 @@ functions:
   - name: 'sum'
     nullability: DECLARED_OUTPUT
     description: "Sum a set of values."
-    options:
-      overflow: [SILENT, SATURATE, ERROR]
     variants:
       - name: scalar
         decomposable: MANY
         parameters:
-          - K: [i8,i16,i32,i64,fp32,fp64]
+          - K: {types: [i8,i16,i32,i64,fp32,fp64]}
+          - overflow:
+            options: [SILENT, SATURATE, ERROR]
+            required: false
```

Comment:
Really minor nit: This is legal YAML but for consistency maybe pick one style and stick with it:

```
          - K:
            types: [i8,i16,i32,i64,fp32,fp64]
          - overflow:
            options: [SILENT, SATURATE, ERROR]
            required: false
```

...or...
```
          - K: {types: [i8,i16,i32,i64,fp32,fp64]}
          - overflow: {options: [SILENT, SATURATE, ERROR], required: false}
```

Reply:
this really made me laugh :)

it made more sense before I started introducing the templating thing...